app/src/main/res/xml/analytics_global_config.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<resources>
```

Comment:
I added these attributes to the code:

https://github.com/nightscout/android-uploader/blob/76-kl.mqttui/app/src/main/java/com/nightscout/android/Nightscout.java#L71-L73